THIRD-PARTY-NOTICES

```diff
@@ -21,10 +21,12 @@ This project includes:
   AutoService under Apache 2.0
   AutoService Processor under Apache 2.0
   Bean Validation API under The Apache Software License, Version 2.0
+  Bouncy Castle Provider under Bouncy Castle Licence
   commonmark-java core under BSD 2-Clause License
   Commons CLI under The Apache Software License, Version 2.0
   Commons IO under The Apache Software License, Version 2.0
   FindBugs-jsr305 under The Apache Software License, Version 2.0
+  IntelliJ IDEA Annotations under The Apache Software License, Version 2.0
```

Comment:
Where did this get pulled in from?

Reply:
No clue..

Reply:
running `mvn dependency:tree` will show you

Reply:
Sure.. removed the IntelliJ entry.